src/main/java/com/google/sps/classes/utilityClass.java

```diff
@@ -0,0 +1,31 @@
+ 
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.classes;
+
+import com.google.gson.Gson;
+
+/**
+ * Utility methods used across classes. Just import class to access all methods.
+ */
+public final class UtilityClass {
```

Comment:
nit: when naming a Class, no need to include 'Class' in the name - readers will understand that it is a classname without it. So in this instance, Utility or UtilityMethods is fine. If you wanted to specify, you could always do MinternUtilities or something similar. Up to you what you decide, but including 'Class' isn't necessary!

Reply:
Done!